src/core/opamStd.mli

```diff
@@ -511,10 +511,12 @@ module Sys : sig
   val os: unit -> os
 
   (** The output of the command "uname", with the given argument. Memoised. *)
-  val uname: string -> string option
+  val uname_cmd: string -> string option
 
   (** The output of the command "getconf", with the given argument. Memoised. *)
   val getconf: string -> string option
+  (** The memoized result of uname(2) *)
+  val uname : OpamStubs.uname Lazy.t
```

Comment:
It's better to have a `uname: void -> OpamStubs.uname` to ease readability/writing when used